### PR TITLE
changelog updates for 0.4.0

### PR DESCRIPTION
**Pull request by ehazlett**
Changelog for 0.4.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,25 @@
 Changelog
 ==========
 
+# 0.4.0 (2014-08-11)
```
Comment (thaJeztah) [inline CHANGELOG.md]: 2015 probably ;-)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,25 @@
...
+- Ability to set environment variables in the Docker Engine
+- Internal libmachine updates for stability
+
+## Drivers
+- Google:
+  - Preemptiple instances
+  - Static IP support
+
+## Fixes
+- Swarm Discovery Flag is verifed
```
Comment (nathanleclaire) [inline CHANGELOG.md]: Sp.: verified

File: CHANGELOG.md (hunk 1 above)
Comment (ehazlett) [inline CHANGELOG.md]: Yep.  I'm time traveling.

File: CHANGELOG.md (hunk 2 above)
Comment (ehazlett) [inline CHANGELOG.md]: Nor can I spall.  Thx

**Comment by nathanleclaire** [comment]
Aside from my and @thaJeztah nits, LGTM

**Comment by ehazlett** [comment]
updated

**Pull request merged**